pkg/cmd/root.go

```diff
@@ -1,16 +1,17 @@
 package cmd
 
 import (
-	"github.com/jzelinskie/cobrautil"
+	"github.com/jzelinskie/cobrautil/v2/cobraotel"
+	"github.com/jzelinskie/cobrautil/v2/cobrazerolog"
 	"github.com/spf13/cobra"
 
 	"github.com/authzed/spicedb/pkg/cmd/server"
 	"github.com/authzed/spicedb/pkg/releases"
 )
 
 func RegisterRootFlags(cmd *cobra.Command) {
-	cobrautil.RegisterZeroLogFlags(cmd.PersistentFlags(), "log")
-	cobrautil.RegisterOpenTelemetryFlags(cmd.PersistentFlags(), "otel", cmd.Use)
+	cobrazerolog.New().RegisterFlags(cmd.PersistentFlags())
+	cobraotel.New("spicedb").RegisterFlags(cmd.PersistentFlags())
```

Comment:
I think we might prefer to keep this cmd.Use so that the binary name is not hard coded

Reply:
ah! good catch!